modules/react/combobox/stories/examples/Autocomplete.tsx

```diff
@@ -95,12 +110,12 @@ export const Autocomplete = () => {
         <InputGroup>
           <InputGroup.Input as={FormField.Input.as(AutoCompleteInput)} />
           <InputGroup.InnerEnd
+            cs={styleOverrides.inputGroupInner}
             pointerEvents="none"
-            style={{opacity: loader.isLoading ? 1 : 0, transition: 'opacity 100ms ease'}}
-            width={20}
+            style={{opacity: loader.isLoading ? 1 : 0}}
```

Comment:
should this line use system opacity tokens?

Reply:
Yeah I went ahead and updated